plugins/com.google.cloud.tools.eclipse.util/plugin.properties

```diff
@@ -0,0 +1,2 @@
+pluginName=Cloud Tools for Eclipse Common Utilities
+providerName=Google Inc.
```

Comment:
Since we put this info here, we should update its `MANIFEST.MF` to use these values instead of duplicating them:

```
Bundle-Name: %pluginName%
...
Bundle-Vendor: %providerName%
```

Reply:
Done